lib/skywalking/client.lua

```diff
@@ -131,6 +133,36 @@ function Client:ping(metadata_buffer, backend_http_uri)
     end
 end
 
+-- Send segemnts data to backend
+local function sendSegments(segmentTransform, count, backend_http_uri)
```

Comment:
Why need `count` as input and output, but do nothing?

Reply:
For now, yes. I will change this function return boolean type tomorrow.